coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/metadata/sql/PostgresDatasetMap.scala

```diff
@@ -374,6 +374,25 @@ class PostgresDatasetMapReader[CT](val conn: Connection, tns: TypeNamespace[CT],
     }
   }
 
+  def datasetInfoByResourceNameQuery = "SELECT system_id, next_counter_value, locale_name, obfuscation_key, resource_name FROM dataset_map WHERE resource_name = ?"
```

Comment:
Ah, right, yeah.  soda-fountain definitely has both a resource_name and a resource_name_casefolded column so it can do that.